pkg/apis/enterprise/v1alpha3/indexercluster_types.go

```diff
@@ -55,6 +58,9 @@ type IndexerClusterMemberStatus struct {
 
 	// Flag indicating if this peer belongs to the current committed generation and is searchable.
 	Searchable bool `json:"is_searchable"`
+
+	// Splunk Smart store configuration
+	SmartStore SmartStoreSpec `json:"smartstore,ommitempty"`
```

Comment:
why do we need this in member status?

Reply:
I thought we can use this to check if there is a change in the smartstore config from last reconcile iteration to the now, so that, we can create a new config map. 
Please correct me, if my understanding is wrong in using this way, and if there needs to be a better alternative way.

Reply:
Ahh ok, that makes sense. Generally, it's better to compare to the state of the output. For example, most things we compare to the current spec of the corresponding StatefulSet objects that are produced. In this case, I imagine the "output" would be the ConfigMap that is the app for indexes.conf.

Comparing to indexes.conf seems overly cumbersome. An alternative may be to store this as an extra field in the ConfigMap. The approach of storing it in status is probably fine, though.

Reply:
Will evaluate and take a direction as part of CSPL-348, which is in progress for createing the configmap.